modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -29,12 +29,40 @@ Use this option if you created your API outside API Manager, but want to apply p
 Use this option if you want to provide additional security for your API. 
 +
 . Based on the type of endpoint you are creating, specify the application type for a basic endpoint or the proxy deployment target for an endpoint with proxy.
-. If you want to manage the API using Mule 4 or later, select the *Mule version* checkbox.
-> In *Implementation URI*, replace the example by entering the URL of the site that hosts the API.
+. If you want to manage the API using Mule 4 or later, perform the following steps:
```

Comment:
After seeing how complex the task becomes, with bullet lists later, consider separating Mule 3 from Mule 4 tasks.

Reply:
Done! Thanks!